tests/test_sensor.py

```
@@ -0,0 +1,93 @@
+import pytest
+
+from custom_components.resmed_myair.sensor import (
+    MyAirDeviceSensor,
+    MyAirFriendlyUsageTime,
+    MyAirMostRecentSleepDate,
+    MyAirSleepRecordSensor,
+    SensorEntityDescription,
+)
+
+
+class DummyCoordinator:
```

Ruff (0.12.2)

Missing docstring in public class

(D101)